CLOUDP-228406: [cli] Rename teams (#2601)
mongodb/mongodb-atlas-cli: docs/atlascli/command/atlas-teams-rename.txt

.. _atlas-teams-rename:

==================
atlas teams rename
==================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

Rename a team in your organization.

> To use this command, you must authenticate with a user account or an API key that has the Organization Owner role.

Reviewer: ```suggestion
To use this command, you must authenticate with a user account or have an API key with the Organization Owner role.
```

Syntax
------

.. code-block::
   :caption: Command Syntax

   atlas teams rename <newName> [options]

.. Code end marker, please don't delete this comment

Arguments
---------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - newName
     - string
     - true
     - New label that identifies the team.

Options
-------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - -h, --help
     - 
     - false
     - help for rename
   * - --orgId
     - string
     - false
     - Organization ID to use. This option overrides the settings in the configuration file or environment variable.
   * - -o, --output
     - string
     - false

>      - Output format. Valid values are json, json-path, go-template, or go-template-file. To see full output, use the -o json option.

Reviewer: ```suggestion
     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see the full output, use the -o json option.
```
